docs/source/modules.rst

```diff
@@ -7,16 +7,14 @@ Modules
 
 Modules can be used from the command line or in a playbook task. Ansible
 executes each module on the remote target node, and collects return
-values.
-
-While different modules perform different tasks, their interfaces and responses
-follow similar patterns.
+values. While different modules perform different tasks, their interfaces and
+responses follow similar patterns.
 
 Module reference
 ----------------
 
-Reference material for each module contains documentation on what parameters
-certain modules accept and what values they expect those parameters to be.
+Reference material contains documentation on the accepted parameters and their
+expected values for each module.
```

Comment:
reference documentation is available for certain modules or all of them? it's unclear

Reply:
Every module we author has reference doc, its a requirement and not a choice.